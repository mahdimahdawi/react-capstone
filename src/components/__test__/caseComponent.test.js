import renderer from 'react-test-renderer';
import CaseComponent from '../caseComponent';
import { Provider } from 'react-redux';
import store from '../../redux/cinfigureStore';

test('Render <CaseComponet /> component', () => {
  const comp = renderer.create(
    <Provider store={store}>
      <CaseComponent cases={[]} value={[]}/>
    </Provider>
  ).toJSON();
  expect(comp).toMatchSnapshot();
});