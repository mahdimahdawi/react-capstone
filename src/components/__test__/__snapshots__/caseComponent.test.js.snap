// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render <CaseComponet /> component 1`] = `
<div
  className="list-page"
>
  <h1
    className="page-heading"
  >
    COVID-19 CASES
  </h1>
  <ul
    className="card"
  />
</div>
`;
